Dockerfile

```
@@ -0,0 +1,46 @@
+FROM golang:1.18-buster AS build-setup
+
+RUN apt-get update
+RUN apt-get -y install cmake zip sudo git
+
+ENV FLOW_GO_REPO="https://github.com/onflow/flow-go"
+ENV FLOW_GO_BRANCH=v0.26.16
+
+RUN mkdir /dps /docker /flow-go
+
+WORKDIR /dps
+
+# clone repos
+ADD . /dps
```

unless you're using some of the special features of `ADD`, it's a [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) to use `COPY`
```suggestion
COPY . /dps
```